tests/test_storm_control.py

```diff
@@ -0,0 +1,409 @@
+from swsscommon import swsscommon
+import os
+import sys
+import time
+import json
+from distutils.version import StrictVersion
+import pytest
+
+class TestStormControl(object):
```

Comment:
Every testcase needs to be able to run independently. Please make sure all the configs added during a test are cleaned up at the end of that test

Reply:
Will update the test code.